mlir/lib/Conversion/VectorToArmSME/VectorToArmSME.cpp

```diff
@@ -242,27 +247,25 @@ struct ConstantOpToArmSMELowering : public OpRewritePattern<arith::ConstantOp> {
 
     // Lower non-zero constants to a loop of 'arm_sme.move_vector_to_tile_slice'
     // ops that broadcast the constant to each tile slice.
-    OpBuilder::InsertionGuard g(rewriter);
     auto loc = constantOp.getLoc();
 
     // Unpack 1-d vector type from 2-d vector type.
```

Comment:
[nit] This is somewhat unrelated to this change, but it would be nice if this comment covered "why" (all for we need for SME is 1-d row/column from the constant matrix that we will move to very row/column of the output) rather than "what" (get 1-d vector from a 2-d vector).

Reply:
I've rewrite this comment :+1: